clients/sidesail/lib/rpc/rpc_testchain.dart

```diff
@@ -271,6 +272,15 @@ class TestchainRPCLive extends TestchainRPC {
     final confirmedFut = await _client().call('getblockchaininfo');
     return BlockchainInfo.fromMap(confirmedFut);
   }
+
+  @override
+  Future<void> initBinary(
+    BuildContext context, {
+    List<String>? arg,
+    String? mnemonicPath,
+  }) async {
+    await super.initBinary(context, arg: arg, mnemonicPath: mnemonicPath);
+  }
```

Comment:
I would prefer if this didn't take in a new args `mnemonicPath`. That's what the `List<String>? arg,` is for.

Pass this to the initBinary function instead, making it

```
      case Thunder():
        await bitnamesRPC.initBinary(
          context,
          arg: ['--mnemonic-seed-phrase-path', binary.mnemonicSeedPhrasePath!],
        );

      case Bitnames():
        await bitnamesRPC.initBinary(
          context,
          arg: ['--mnemonic-seed-phrase-path', binary.mnemonicSeedPhrasePath!],
        );

```